MinecraftClient/McClient.cs

```diff
@@ -2125,6 +2125,550 @@ public bool DoWindowAction(int windowId, int slotId, WindowActionType action)
                             }
                         }
                         break;
+                    case WindowActionType.ShiftRightClick:

```

Comment:
code under this is complete same code under shiftclick it's just same like i said. only difference of them what reported to server and when it's needed by some plugin to do action it works.